### PR TITLE
Fix DNS: copy /etc files and add a dhcpcd hook for dynamic DNS config

### PR DESCRIPTION
**Pull request by sylane**
* Copy files and directories from grisp/default/common/deploy/files/etc into /etc
* Add default hosts files
* Add default resolve.conf file that uses dns.sb DNS servers
* Add default erl_inetrc file that point to hosts and resolve.conf
* Fix compilation warnings
* Fix RTEMS shell command configuration
* Add rtems log priority option to ini file

### CODE REVIEW

File: grisp/default/common/deploy/files/erl_inetrc
Comment (ziopio) [inline grisp/default/common/deploy/files/erl_inetrc:1]: So in teory this change will avoid the need for our users to define an erl_inetrc file.
This also means 
- we do not need to generate such file in the `rebar3 gris configure` plugin.
- grisp_connect must NOT have an overlay for this file
-  we need to update the wiky
- make sure we do new releases of all related projects

**Comment by ziopio** [comment]
I rebased the branch

**Comment by sylane** [comment]
Should we merge this and then your PR about DHCP DNS, or should we merge them somehow ?

**Comment by ziopio** [comment]
When we are good we merge my PR into this branch

**Review by ziopio — APPROVED** [review]
Just update the changelog

**Pull request merged**